test/ReactWrapper-spec.jsx

```diff
@@ -3005,5 +3005,18 @@ describeWithDOM('mount', () => {
         expect(wrapper.name()).to.equal('div');
       });
     });
+
+    describe('unavailable ref', () => {
+      it('should return empty nodes', () => {
+        class WithoutRef extends React.Component {
+          render() { return <div />; }
+        }
+        const wrapper = mount(<WithoutRef />);
+        const ref = wrapper.ref('not-a-ref');
+
+        expect(ref.length).to.equal(0);
+        expect(ref.isEmpty()).to.equal(true);
+      });
+    });
```

Comment:
Can you name the top level `describe` block `.ref()` so we can put future tests for `ref` in there too? Thanks!